xbmc/music/GUIViewStateMusic.cpp

```diff
@@ -193,6 +193,8 @@ CGUIViewStateMusicDatabase::CGUIViewStateMusicDatabase(const CFileItemList& item
       AddSortMethod(SortByYear, 562, LABEL_MASKS("%F", "", strAlbumLeft, strAlbumRight));
       // album date added
       AddSortMethod(SortByDateAdded, sortAttribute, 570, LABEL_MASKS("%F", "", strAlbumLeft, strAlbumRight));  // Filename, empty | Userdefined, Userdefined
+      // play count
+      AddSortMethod(SortByPlaycount, 567, LABEL_MASKS("%F", "", strAlbumLeft + " - " + strAlbumRight, "%V"));  // Filename, empty | Userdefined - Userdefined, Play count
```

Comment:
Once again I'd like to point out that we should stick to the old way of putting together the label mask until it has been decided how to handle this for all affected sort methods.